canonical/lxd-ui
# feat: right align bottom buttons with forms across app [WD-7985]

## Done

- adjusted all creation and edit forms to have max width of 54rem
- right aligned bottom control buttons to all creation and edit form contents

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - Check all forms with bottom control buttons. Make sure that buttons are right aligned to the form content for all screen sizes and also check if the forms visually look okay.

## Screenshots

Example
![Screenshot from 2024-01-05 11-25-49](https://github.com/canonical/lxd-ui/assets/151511568/d7d7d99c-9472-4423-87db-8edae1f23d57)
